release_docs/RELEASE.txt

```diff
@@ -544,14 +546,14 @@ Bug Fixes since HDF5-1.12.2 release
 
       Fixes GitHub issue #2682
```

Comment:
The note for this issue 2682 needs fixed:
    - Added option --no-compact-subset to h5diff

      When the name of an object contains square brackets, h5diff was not able
      to parse it correctly because “[ ]” were interpreted as the compact form
      of subsetting.

      h5dump has the special argument, --no-compact-subset to allow the use of
      ‘[‘ and ‘]’ in dataset names.  h5diff now has this option as well.

      Fixes GitHub issue #2682

Reply:
Is this replacement text for lines 535-535, i.e. replaces 

>       The install dependencies were not including the need for MPI libraries when
      an application or library was built with the C library. Also updated the
> Fixes GitHub issue #2682

Reply:
There is a text expansion above Fixes GitHub issue #2682